taskcluster/darwin-opt-base.tyml

```diff
@@ -42,17 +42,15 @@ payload:
       - "-cxe"
       - >
         export TASKCLUSTER_ARTIFACTS="$(pwd)/public/" &&
-        export TASKCLUSTER_TASK_ROOT="$(pwd)" &&
-        export TASKCLUSTER_TASK_DIR="$(dirname `pwd`)/task-${event.head.sha}" &&
+        export TASKCLUSTER_TASK_DIR="$(pwd)" &&
```

Comment:
Cleanup work, we cannot rely anymore on that kind of "task-SHA1" path if we have multiple workers anyway.